providers/ebs_raid.rb

```diff
@@ -390,9 +390,12 @@ def create_raid_disks(mount_point, mount_point_owner, mount_point_group, mount_p
         case filesystem
           when "ext4"
             system("mke2fs -t #{filesystem} -F #{md_device}")
+          when "xfs"
+            run_context.include_recipe 'xfs'
+            system("mkfs -t #{filesystem} #{md_device}")
```

Comment:
`mkfs.xfs #{md_device}` is how I normally setup the FS, so I'll test this with my change and resubmit the PR assuming everything works.